ApiDoctor.Console/App.config

```diff
@@ -1,8 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <configuration>
-    <configSections>
-    </configSections>
-    <startup> 
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" />
-    </startup>
+  <runtime>
+    <gcServer enabled="true"/>
```

Comment:
why specify the gc type?

Reply:
This was an investigation into improving performance. By default it uses workstation gc. I will remove this pending further investigation as it did not bear fruit